modules/oadp-configuring-aws-bsl.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-aws-bsl.adoc_{context}"]
+= Configuring the backup storage location using AWS
+
+You can configure the {aws-short} backup storage location (BSL) as shown in the following example.
+
+.Prerequisites
+
+* You have created an object storage bucket using {aws-short}.
+* You have installed the {oadp-short} Operator.
+
+.Procedure
+
+. Configure the BSL custom resource (CR) as shown in the following example:
++
+.Example backup storage location
```

Comment:
This generally looks good, but I think one issue is that there are multiple references to this being an example, while this yaml file has user-replaced values instead of actual example values. I would either ask an SME for example values, or change the wording in this line and in the step to not refer to this as an example

Reply:
re-worded it. thanks.